docs/resources/tensorflow_frontend.md

```diff
@@ -2,16 +2,14 @@
 
 @sphinxdirective
 
-TensorFlow Frontend is C++ based Frontend for conversion of TensorFlow models and is available as a preview feature starting from 2022.3.
-That means that you can start experimenting with ``--use_new_frontend`` option passed to Model Optimizer to enjoy improved conversion time for limited scope of models
-or directly loading TensorFlow models through ``read_model()`` method.
+TensorFlow Frontend is C++ based Frontend for conversion of TensorFlow models. MO now uses the TensorFlow Frontend as the default path for conversion to IR.
+Also, the frontend allows to load TensorFlow models (in SavedModel, MetaGraph, and frozen Protobuf formats) directly via ``read_model()`` method.
 
 The current limitations:
 
 * IRs generated by new TensorFlow Frontend are compatible only with OpenVINO API 2.0
-* There is no full parity yet between legacy Model Optimizer TensorFlow Frontend and new TensorFlow Frontend so primary path for model conversion is still legacy frontend
-* Model coverage and performance is continuously improving so some conversion phase failures, performance and accuracy issues might occur in case model is not yet covered.
-Known unsupported models: object detection models and all models with transformation configs, models with TF1/TF2 control flow, Complex type and training parts
-* ``read_model()`` method supports only ``*.pb`` format while Model Optimizer (or ``convert_model`` call) will accept other formats as well which are accepted by existing legacy frontend
+* There is no full parity between the legacy frontend and new frontend in MO. Known limitations compared to the legacy approach are:
+TF1 Control flow, Complex types, models requiring config files and old python extensions. The solution detects unsupported functionalities and provides fallback.
+To force using the legacy frontend, ``--use_legacy_frontend`` must be specified.
```

Comment:
Maybe "to force the use of the legacy fronted"? Force using gives an impression like force something with the help of the frontend

Reply:
Fixed.